initBookshelf.js

```diff
@@ -1,9 +1,15 @@
 const instances = new Map();
 
-module.exports = (knex, plugins = ['registry', 'bookshelf-camelcase', 'visibility', 'pagination', 'bookshelf-paranoia']) => {
+module.exports = (knex, plugins = ['registry', 'bookshelf-camelcase', 'visibility', 'pagination', 'bookshelf-paranoia', 'bookshelf-cursor-pagination']) => {
```

Comment:
If it is not a default plugin and we dont want to support it, I would let the developer to decide the set of plugins.

Changes in this file are not necessary.